fdbserver/worker.actor.cpp

```diff
@@ -753,6 +758,37 @@ ACTOR Future<Void> monitorServerDBInfo( Reference<AsyncVar<Optional<ClusterContr
 	}
 }
 
+class SharedLogsKey {
+	TLogVersion logVersion;
+	TLogSpillType spillType;
+	KeyValueStoreType storeType;
+
+public:
+	SharedLogsKey( const TLogOptions& options, KeyValueStoreType kvst ) {
+		logVersion = options.version;
+		spillType = options.spillType;
+		storeType = kvst;
```

Comment:
I prefer to initialize members in the initialization list, which is a little more efficient.